event-streams/src/main/java/com/redhat/cloud/platform/playbook_dispatcher/RunEventTransform.java

```diff
@@ -143,6 +145,31 @@ Payload buildRunPayload(Struct input) {
         payload.setCreatedAt(input.getString("created_at"));
         payload.setUpdatedAt(input.getString("updated_at"));
 
+        // If the option fields do not exist, set them to empty strings
+        try {
+            payload.setName(input.getString("name"));
+        } catch (DataException e) {
+            payload.setName("");
+        }
+
+        try {
+            payload.setWebConsoleUrl(URI.create(input.getString("web_console_url")));
+        } catch (DataException e) {
+            payload.setWebConsoleUrl(URI.create(""));
+        }
+
+        RecipientConfig recipientConfig;
+        try {
+            recipientConfig = this.objectMapper.readValue(input.getString("recipient_config"), RecipientConfig.class);
```

Comment:
Recipient_config is not an actual field in the database. You'll need to construct it by using the sat_id and sat_org_id fields from. https://github.com/RedHatInsights/playbook-dispatcher/blob/master/migrations/011_alter_runs_sat-fields.up.sql#L5